Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node
+FROM node:dubnium-alpine
```

Comment:
Why not use `node:11.12.0-alpine` instead of `node:dubnium-alpine`? That's not the latest version and regular version numbers are easier to read without prior knowledge than named versions.